pkg/ceph/client/pool.go

```diff
@@ -129,20 +130,33 @@ func CreatePool(context *clusterd.Context, clusterName string, newPool CephStora
 }
 
 func CreatePoolForApp(context *clusterd.Context, clusterName string, newPool CephStoragePoolDetails, appName string) error {
-	args := []string{"osd", "pool", "create", newPool.Name, strconv.Itoa(newPool.Number)}
+	// create a crush rule for a replicated pool, if a failure domain is specified
+	replicated := newPool.ErasureCodeProfile == "" && newPool.Size > 0
+	ruleName := fmt.Sprintf("%s-rule", newPool.Name)
+	if replicated && newPool.FailureDomain != "" {
+		args := []string{"osd", "crush", "rule", "create-simple", ruleName, "default", newPool.FailureDomain}
```

Comment:
what exactly is linking this rule to the specific pool?  is it implicit because the pool name is in the rule name?

Reply:
on line 150 the rule name is passed to the pool creation. I'll add a comment